src/nodedef.cpp

```diff
@@ -1272,15 +1283,22 @@ void ContentFeatures::deSerializeOld(std::istream &is, int version)
 			groups[name] = value;
 		}
 		drawtype = (enum NodeDrawType)readU8(is);
+		bool ignore_culling = false;
+		if ((version <= 26) && ((drawtype == NDT_MESH) ||
```

Comment:
Same there.
Another point: bool ignore_culling = (result of your huge condition) is better :)